docs/cocoapods-setup.md

```diff
@@ -9,14 +9,16 @@ setup process is fairly straightforward and typical.
 using EarlGrey 2.0 with.
 
 Here is an extremely simple Podfile demonstrating integrating an app called
-`App` with EarlGrey 2.0.
+`App` with EarlGrey 2.0. Note that you must pin the transitive dependency
+`eDistantObject` at 0.9.0.
```

Comment:
Here is an extremely simple Podfile demonstrating integrating an app called
`App` with EarlGrey 2.0.

**Note:**  that you must pin the transitive dependency `eDistantObject` at
0.9.0.